signup/tool/src/webapp/signup/signupMeetings.jsp

```diff
@@ -227,7 +227,11 @@
 						<h:outputText value="&nbsp;" escape="false" rendered="#{!SignupMeetingsBean.enableExpandOption}"/>
 					</h:panelGroup>
 				</h:panelGrid>
-				
+
+				<h:panelGrid columns="1">
+					<h:outputText value="&nbsp;" escape="false"/>
```

Comment:
I think this should be accomlished via CSS rather than inserting non-breaking spaces.